lib/android/app/src/main/java/com/reactnativenavigation/parse/NavigationOptions.java

```diff
@@ -32,15 +40,28 @@ public static NavigationOptions parse(JSONObject json, @NonNull NavigationOption
 		result.topBarOptions = TopBarOptions.parse(json.optJSONObject("topBar"));
 		result.bottomTabsOptions = BottomTabsOptions.parse(json.optJSONObject("tabBar"));
 
+		result.leftButtons = Button.parseJsonArray(json.optJSONArray("leftButtons"));
+		result.rightButtons =  Button.parseJsonArray(json.optJSONArray("rightButtons"));
+
 		return result.withDefaultOptions(defaultOptions);
 	}
 
 	public TopBarOptions topBarOptions = new TopBarOptions();
 	public BottomTabsOptions bottomTabsOptions = new BottomTabsOptions();
+	public ArrayList<Button> leftButtons;
+	public ArrayList<Button> rightButtons;
 
 	public void mergeWith(final NavigationOptions other) {
         topBarOptions.mergeWith(other.topBarOptions);
 		bottomTabsOptions.mergeWith(other.bottomTabsOptions);
+
+		if(other.leftButtons != null) {
+			leftButtons = other.leftButtons;
```

Comment:
you need to merge button options. Because now it's replaced. So, if you want just to change one property, you need to provide all another once more.